config/routes.rb

```diff
@@ -2,13 +2,13 @@
   root 'home#index'
 
   resources :sessions, only: [:create]
-  resources :recommendations, only: [:index,  :create, :update, :destroy]
+  resources :recommendations, only: [:create, :update, :destroy]
 
-  resources :watched_recommendations, only: [:destroy]
+  get '/recommendations', to: 'recommendations#index', watched_status: false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.